Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Editor/CalibrationTestWindow.cs

```diff
@@ -0,0 +1,521 @@
+                    if (CalculatedCameraCalibration.TryDeserialize(calibrationDataPayload, out calibration))
+                    {
+                        calibrationDataForPlayback = new CalibrationData(calibration.Intrinsics, calibration.Extrinsics);
+
+                        isCalibrationDataParsed = true;
+                    }
+                }
+            }
+            catch
+            {
+            }
+        }
+
+        private void Instance_TextureRenderCompleted()
+        {
+            if (Time.time > nextRecordingFrameTime)
+            {
+                CompositionManager compositionManager = GetCompositionManager();
+
+                string rootFolder, targetFolder;
+                GetRecordingDirectories(out rootFolder, out targetFolder);
+                if (!Directory.Exists(rootFolder))
+                {
+                    Directory.CreateDirectory(rootFolder);
+                }
+                if (!Directory.Exists(targetFolder))
+                {
+                    Directory.CreateDirectory(targetFolder);
+                }
+
+                string fileName = Path.Combine(targetFolder, $"{recordingForRecording.Poses.Count}.raw");
+
+                UnityCompositorInterface.TakeRawPicture(fileName);
+
+                recordingForRecording.Poses.Add(new CalibrationRecordingPose
+                {
+                    FrameFileNumber = recordingForRecording.Poses.Count,
+                    FrameTime = Time.time,
+                    CameraPosition = compositionManager.transform.parent.localPosition,
+                    CameraRotationEuler = compositionManager.transform.parent.localRotation.eulerAngles,
+                });
+
+                nextRecordingFrameTime += 1.0f / 10.0f;
+            }
+        }
+
+        private void GetRecordingDirectories(out string rootFolder, out string targetFolder)
+        {
+            rootFolder = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "CalibrationVideos");
+            targetFolder = Path.Combine(rootFolder, currentRecordingSubdirectoryName);
+        }
+
+        private void StartPlayback()
+        {
+            currentTime = 0;
+            currentFrameIndex = 0;
+            recordingForPlayback = JsonUtility.FromJson<CalibrationRecording>(File.ReadAllText(indexFilePath));
+            ApplyFrame(currentFrameIndex);
+
+            CompositionManager compositionManager = GetCompositionManager();
+            HolographicCameraNetworkManager networkManager = GetHolographicCameraNetworkManager();
+
+            previousCalibrationData = compositionManager.CalibrationData;
+            compositionManager.EnableHolographicCamera(networkManager.transform, calibrationDataForPlayback);
+
+            testCube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            testCube.transform.localScale = Vector3.one * 0.1f;
+            testCube.transform.localPosition = new Vector3(0.0f, 0.0f, 0.05f);
```

Comment:
Should this be configurable in the window based on marker size?

Reply:
Tied this to the constant defined for the size of the ArUco marker